packages/grafana-ui/src/components/Table/utils.test.tsx

```diff
@@ -474,7 +500,7 @@ describe('Table utils', () => {
       expect(sortNumber(a, b, '0')).toEqual(expected);
     });
 
-    it.skip('should have good performance', () => {
+    it('should have good performance', () => {
```

Comment:
I think this needs to be reverted. It is failing the CI build

Reply:
skipped. but I would like to know why this kills the build...

Reply:
I assume the duration of loading some table under some circumstances detailed in the test is larger than the expected value and thus it fails:

```
FAIL packages/grafana-ui/src/components/Table/utils.test.tsx (6.321 s)
  ● Table utils › sortNumber › should have good performance

    expect(received).toBeLessThanOrEqual(expected)

    Expected: <= 20
    Received:    53.36573200020939

      516 |       const stop = performance.now();
      517 |       const diff = stop - start;
    > 518 |       expect(diff).toBeLessThanOrEqual(20);
          |                    ^
```